### PR TITLE
EOS-28480: commands needed by motr support bundle, but not present in…

### PR DESCRIPTION
**Pull request by papan-singh**
### Specification of requested package

Details | Values
------ | ------
Package Name  |  file-5.11-37.el7.x86_64
Version | 5.11
license | BSD
Source  | 

### Please add below details about package

* [ ] Purpose of using new SW ?
motr support bundle 
* [ ] Location of the source code of new opensource SW ?
 http://buildlogs-seed.centos.org/c7.2009.00.x86_64/file/20200930160602/5.11-37.el7.x86_64/
* [ ] Did you evaluate existing SW and confirm they do not serve the purpose ? If yes, include the comparison. If no, why ?
No any command which can be used instead of file 
* [ ] Who is building it or how it gets built ? If yes, Which component/rpm will it be included ? i.e. name of the rpm which includes SW.
file-5.11-37.el7.x86_64
* [ ] Download location of opensource software package ?
http://buildlogs-seed.centos.org/c7.2009.00.x86_64/file/20200930160602/5.11-37.el7.x86_64/
* [ ] Who will be supporting this package ?
* [ ] Is there a configuration required for this? Who will be configuring it ?
* [ ] Which component is responsible for deploying on nodes ?
* [ ] What is the licensing policy ? Is it approved by legal ?

### CODE REVIEW

**Comment by shailesh-vaidya** [comment]
@papan-singh  - Please update template with required details for third party packages to be added.

File: docker/cortx-deploy/third-party-rpms.txt [hunk 1]
```diff
@@ -20,5 +20,7 @@ python36-pika-0.10.0 #cortx-s3srever
 hiredis-0.12.1 #cortx-s3srever
 sshpass
 sudo
+file
```
Comment (shailesh-vaidya) [inline docker/cortx-deploy/third-party-rpms.txt:23]: Please add component name infront of package name. 
```
file #cortx-motr
less #cortx-motr
```

Comment (papan-singh) [reply docker/cortx-deploy/third-party-rpms.txt]: incorporated

**Comment by shailesh-vaidya** [comment]
@ujjwalpl @johnbent  - Could you please review this request of adding packages to enable `file` and `less` commands in cortx-all image.

**Review by ujjwalpl — APPROVED** [review]

**Comment by shailesh-vaidya** [comment]
@johnbent  - Could you please review and approve this PR

**Review by shailesh-vaidya — APPROVED** [review]

**Review by ujjwalpl — APPROVED** [review]

**Review by johnbent — APPROVED** [review]

**Pull request merged**